py/selenium/webdriver/remote/webdriver.py

```diff
@@ -82,11 +87,11 @@ def __init__(self, command_executor='http://127.0.0.1:4444/wd/hub',
         if type(self.command_executor) is bytes or isinstance(self.command_executor, str):
             self.command_executor = RemoteConnection(command_executor, keep_alive=keep_alive)
         self._is_remote = True
-        self.session_id = None
-        self.capabilities = {}
+        self.session_id = session_id
+        self.capabilities = capabilities
         self.error_handler = ErrorHandler()
         self.start_client()
-        self.start_session(desired_capabilities, browser_profile)
+        self.start_session(desired_capabilities, browser_profile, force_new=False)
```

Comment:
Hi, are you waiting for my feedback, or am i waiting for your feedback? :)

Bests!